widget/form.go

```diff
@@ -24,9 +24,11 @@ func NewFormItem(text string, widget fyne.CanvasObject) *FormItem {
 type Form struct {
 	BaseWidget
 
-	Items    []*FormItem
-	OnSubmit func()
-	OnCancel func()
+	Items        []*FormItem
+	SubmitButton *Button
```

Comment:
I think your idea of setting text worked more in-line with how other widgets are structured.
That way we control the internal details of how the buttons look and behave, should provide a more consistent UI overall.
And shorter code for people wanting to change the text :)
Apologies that this was not clear the way the previous review was worded.

Reply:
Fixed.